lib/saluki-event/src/eventd/mod.rs

```diff
@@ -1,9 +1,33 @@
-//! Event types
-mod priority;
-pub use self::priority::*;
+//! Events.
 
-mod alert;
-pub use self::alert::*;
+// TODO: Switch usages of `String` to `MetaString` since we should generally be able to intern these strings as they
+// originate in in the DogStatsD codec, where interning is already taking place.
+
+/// Alert type.
+#[derive(Clone, Copy, Debug)]
+pub enum AlertType {
+    /// Indicates an informational event.
+    Info,
+
+    /// Indicates an error event.
+    Error,
+
+    /// Indicates a warning event.
+    Warning,
+
+    /// Indicates a successful event.
+    Success,
+}
+
+/// Event priority.
+#[derive(Clone, Copy, Debug)]
+pub enum Priority {
```

Comment:
For both `Priority` and `AlertType`, I've moved them to `mod.rs` because they're simple enough that it ends up being a little cleaner to have it all in the same file. If we had an impl block, or tests, then it might make sense to push it into a separate file.